.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always missing this feature when [...]
```

Comment:
I found this section confusing. If there is a problem, there should be a bug fix.

Reply:
Here is if the feature is a consequence of a problem, not a bug. Like "I'm requesting this because there is no way to solve this problem at this moment."

I will be glad to add any suggestions here.

Reply:
Okay, I got the idea. Let me think of a suggestion to make it clear and get back to you.